lib/ui/painting.dart

```diff
@@ -10,12 +10,22 @@ Color _scaleAlpha(Color a, double factor) {
 
 /// An immutable 32 bit color value in ARGB
 class Color {
-  /// Construct a color from the lower 32 bits of an int.
+  /// Construct a color from the lower 32 bits of an [int].
   ///
-  /// Bits 24-31 are the alpha value.
-  /// Bits 16-23 are the red value.
-  /// Bits 8-15 are the green value.
-  /// Bits 0-7 are the blue value.
+  /// The bits are interpreted as follows:
+  ///
+  /// * Bits 24-31 are the alpha value.
+  /// * Bits 16-23 are the red value.
+  /// * Bits 8-15 are the green value.
+  /// * Bits 0-7 are the blue value.
+  ///
+  /// In other words, if AA is the alpha value in hex, RR the red value in hex,
+  /// GG the green value in hex, and BB the blue value in hex, a color can be
+  /// expressed as `const Color(0xAARRGGBB)`.
+  ///
+  /// For example, to get a fully opaque orange, you would use `const
+  /// Color(0xFFFF9000)` (`FF` for the alpha, `FF` for the red, `90` for the
+  /// green, and `00` for the blue).
```

Comment:
You might want to give a specific example of translating `#FF9000` into this format as the `#` form of naming colors is very widely used (e.g., in CSS as well as Photoshop).